lib/matplotlib/tests/test_triangulation.py

```diff
@@ -56,15 +56,12 @@ def test_delaunay():
 
 
 def test_delaunay_duplicate_points():
-    # x[duplicate] == x[duplicate_of]
-    # y[duplicate] == y[duplicate_of]
     npoints = 10
     duplicate = 7
     duplicate_of = 3
 
     np.random.seed(23)
-    x = np.random.random((npoints))
-    y = np.random.random((npoints))
+    x, y = np.random.random((2, npoints))
```

Comment:
Personally, I would leave this as two lines because it's slightly easier to understand.

Reply:
reverted